reducers/gotPlayer/player.js

```diff
@@ -28,3 +31,19 @@ export default (state = initialState, action) => {
       return state;
   }
 };
+
+export const player = {
```

Comment:
Yes but I'd like to have better encapsulation of state so if we update the reducers we don't have to do any refactoring. Just trying this pattern out for now to see how it feels, cause it doesn't feel good that the containers need to know the structure of the store